frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ChangeOverwriteWarning/strategy-change-diff-calculation.ts

```diff
@@ -142,9 +142,31 @@ export function getStrategyChangesThatWouldBeOverwritten(
     change: IChangeRequestUpdateStrategy,
 ): ChangesThatWouldBeOverwritten | null {
     const fallbacks = { segments: [], variants: [], title: '' };
+
+    const withSortedSegments = (() => {
+        if (change.payload.segments && currentStrategyConfig?.segments) {
+            const changeCopy = {
+                ...change,
+                payload: {
+                    ...change.payload,
+                    segments: [...(change.payload.segments ?? [])].sort(),
```

Comment:
Do we still need the `??` since we're truthy checking this on line 147?

Reply:
Nope; good call!